smithy-typescript-codegen/build.gradle.kts

```diff
@@ -18,6 +18,7 @@ extra["displayName"] = "Smithy :: Typescript :: Codegen"
 extra["moduleName"] = "software.amazon.smithy.typescript.codegen"
 
 dependencies {
+    api("software.amazon.smithy:smithy-aws-traits:1.0.5")
```

Comment:
This package should not take dependencies on AWS specific components. If functionality within it needs to change to match usage in the JS v3 SDK, it should be abstracted here so it can be configured there.